client/group.js

```diff
@@ -0,0 +1,38 @@
+Group = function(router, options, parent) {
+  options = options || {};
+  this._router = router;
+  this.prefix = options.prefix || '';
+
+  this._middlewares = options.middlewares || [];
+  this._subscriptions = options.subscriptions || Function.prototype;
+
+  if (parent) {
+    this.prefix = parent.prefix + this.prefix;
+    this._middlewares = parent._middlewares.concat(this._middlewares);
+    this.parent = parent;
+  }
+};
+
+Group.prototype.route = function(path, options) {
+  path = this.prefix + path;
```

Comment:
We need to check this path starts with `/`. Otherwise throw an error.
We may can do this for Router's route method as well.